common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -31,6 +31,9 @@
 from xmodule.util.django import get_current_request_hostname
 import xblock.reference.plugins
 
+from django.dispatch import receiver
```

Comment:
`django.dispatch` is already imported above, do we need this as well?

At any rate, I think the `django.*` imports should all go together in the block above.